test/runtime/configMatters/comm/ugni/overflow-nic-tlb.wrap

```diff
@@ -11,8 +11,6 @@ if [[ -n "$hpm" ]] ; then module unload $hpm ; fi
 
 export CHPL_RT_CALL_STACK_SIZE=128K
 case $($CHPL_HOME/util/chplenv/chpl_platform.py --target) in
```

Comment:
Maybe just remove the `case` statement entirely? Without a `*)` it doesn't seem needed.